tests/integration/kubernetes/k8s-sealed-secret.bats

```diff
@@ -15,7 +15,7 @@ export KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
 export AA_KBC="${AA_KBC:-cc_kbc}"
 
 setup() {
-	[ "${KATA_HYPERVISOR}" = "qemu-coco-dev" ] || skip "Test not ready yet for ${KATA_HYPERVISOR}"
+	[[ "${KATA_HYPERVISOR}" = "qemu-coco-dev" || is_confidential_runtime_class ]] || skip "Test not ready yet for ${KATA_HYPERVISOR}"
```

Comment:
Would:
```
    if ! is_confidential_runtime_class; then
        skip "Test not supported for ${KATA_HYPERVISOR}."
    fi
```
be more readable and do the same?

Reply:
yep, thanks for the tip. I will update the PR.